graylog2-web-interface/src/components/graylog/ListGroupItem.jsx

```diff
@@ -3,80 +3,112 @@ import PropTypes from 'prop-types';
 import styled, { css } from 'styled-components';
 // eslint-disable-next-line no-restricted-imports
 import { ListGroupItem as BootstrapListGroupItem } from 'react-bootstrap';
-import { darken } from 'polished';
+import { darken, lighten } from 'polished';
 
 import { useTheme } from 'theme/GraylogThemeContext';
 import { util } from 'theme';
 import bsStyleThemeVariant from './variants/bsStyle';
 
-const listGroupItemStyles = (hex) => {
+const listGroupItemStyles = (hex, variant) => {
   const backgroundColor = util.colorLevel(hex, -9);
   const textColor = util.colorLevel(hex, 6);
 
   return css`
-    && {
+    &.list-group-item-${variant} {
```

Comment:
Updated styles to use [Bootstrap 3.4.1](https://github.com/twbs/bootstrap/blob/v3.4.1/less/list-group.less)